### PR TITLE
Issue 43496: View names are not really case-insensitive

### PR DESCRIPTION
**Pull request by XingY**
#### Rationale
Custom views aren't case-insensitive but it's treated so on several UI areas.

#### Related Pull Requests
* 

#### Changes
* show all views for grid views dropdown and manage views

### CODE REVIEW

File: query/src/org/labkey/query/QueryServiceImpl.java [hunk 1]
```diff
@@ -779,7 +779,7 @@ protected QueryDefinition getQueryDefMetadataOverride(User user, @NotNull Contai
     protected Map<String, CustomView> getCustomViewMap(@NotNull User user, Container container, @Nullable User owner, QueryDefinition qd,
                                                        boolean inheritable, boolean sharedOnly)
     {
-        Map<String, CustomView> views = new CaseInsensitiveHashMap<>();
+        Map<String, CustomView> views = new HashMap<>();
```
Comment (labkey-nicka) [inline query/src/org/labkey/query/QueryServiceImpl.java:782]: From a convention perspective it seems a bit odd to me that "schemas" and "queries" are case-insensitive but "views" are case-sensitive.  I would not expect this. Could we consider going the other way and preventing the persistence of views with the same name case-insensitively?

**Review by labkey-nicka — COMMENTED** [review]
I'd like to discuss further.

File: query/src/org/labkey/query/QueryServiceImpl.java (hunk 1 above)
Comment (labkey-adam) [reply query/src/org/labkey/query/QueryServiceImpl.java:782]: I tend to agree. We were trying (and apparently failing) to be case-insensitive here. In addition to schema & query consistency, I think case-insensitive will ensure consistency of view naming between PostgreSQL and Microsoft SQL Server.

Comment (XingY) [reply query/src/org/labkey/query/QueryServiceImpl.java:782]: Thanks both. updated ticket and closing this PR

**Pull request closed without merging**